woocommerce/woocommerce-android
# Add Interac refund canceled event on back press while Interac refund is in progress

Closes: #6372 


### Description

This PR adds `*_interact_refund_cancelled` event. This event gets fired up on back press while Interac refund is in progress.

### Testing instructions

1. Collect payment from an order eligible for IPP (store must be setup in Canada)
2. Make sure the payment is successfully collected from the above order.
3. Try to refund using the same Interac card for the above order.
4. While the Interac refund is in progress, press back and check the logs
5. Ensure `*_interact_refund_cancelled` event is triggered with appropriate properties.
